api/src/database/helpers/schema/dialects/mysql.ts

```diff
@@ -15,32 +14,6 @@ export class SchemaHelperMySQL extends SchemaHelper {
 		return getDefaultIndexName(type, collection, fields, { maxLength: 64 });
 	}
 
-	override applyMultiRelationalSort(
```

Comment:
As we are no longer using `applyMultiRelationalSort` for mysql it looks like we can remove it completely from https://github.com/directus/directus/blob/main/api/src/database/helpers/schema/types.ts#L149 ? It does not seem to be a necessary abstraction anymore.

Reply:
Sure thing